src/inc/utilcode.h

```diff
@@ -144,17 +144,25 @@ typedef LPSTR   LPUTF8;
 
 #ifndef DEBUG_NOINLINE
 #if defined(_DEBUG)
+#ifdef _MSC_VER
 #define DEBUG_NOINLINE __declspec(noinline)
 #else
+#define DEBUG_NOINLINE __attribute__((noinline))
+#endif
+#else
 #define DEBUG_NOINLINE
 #endif
 #endif
 
 #ifndef DBG_NOINLINE_X86__RET_INLINE
 #if defined(_DEBUG) && defined(_TARGET_X86_)
 // this exists to make scan work on x86. 
+#ifdef _MSC_VER
 #define DBG_NOINLINE_X86__RET_INLINE __declspec(noinline)
```

Comment:
Actually, I think you can just delete this weird macro. It does not seem to be used anywhere.